utils/swift_build_support/swift_build_support/debug.py

```diff
@@ -30,7 +30,8 @@ def print_xcodebuild_versions(file=sys.stdout):
     version = shell.capture(
         ['xcodebuild', '-version'], dry_run=False, echo=False).rstrip()
     sdks = shell.capture(
-        ['xcodebuild', '-version', '-sdk'], dry_run=False, echo=False).rstrip()
+        ['xcodebuild', '-version', '-sdk'], dry_run=False, echo=False,
+        allow_non_zero_exit=True).rstrip()
```

Comment:
Probably a good idea to have a comment here explaining why this is necessary, so someone doesn't accidentally remove it.